pkg/reconciler/route/route.go

```diff
@@ -276,6 +278,14 @@ func (c *Reconciler) tls(ctx context.Context, host string, r *v1.Route, traffic
 	sort.Slice(acmeChallenges, func(i, j int) bool {
 		return acmeChallenges[i].URL.String() < acmeChallenges[j].URL.String()
 	})
+
+	orphanCerts, err := c.getOrphanRouteCerts(r, domainToTagMap)
+	if err == nil {
```

Comment:
this doesn't handle the error case, is that intentional? Might be better to return an error if there's an error and otherwise do the orphan deletion